### PR TITLE
Added the ability to set your bell from the web app.

### PR DESCRIPTION
**Pull request by sanderDijkxhoorn**

### CODE REVIEW

File: components/Controls.tsx [hunk 1]
```diff
@@ -190,3 +221,25 @@ function SetPowerLevelButton({ level, selected, onSelect }: SetPowerLevelButtonA
...
+            onClick={onSelect}
+            style={{
+                margin: 4,
+                padding: '6px 10px',
+                width: 50,
+                height: 50,
+                backgroundColor: selected ? 'var(--active-button-bg-color)' : undefined,
+            }}
+        >
+            <h1 style={{ margin: 0 }}>{tone}</h1>
```
Comment (mjarkk) [inline components/Controls.tsx:242]: I think we should add a description below this like the speed limit buttons

image

**Review by mjarkk — CHANGES_REQUESTED** [review]

File: components/Controls.tsx (hunk 1 above)
Comment (sanderDijkxhoorn) [reply components/Controls.tsx:242]: Which descriptions should I add?

Boat, bell & party?

Comment (mjarkk) [reply components/Controls.tsx:242]: Sounds good!

Comment (sanderDijkxhoorn) [reply components/Controls.tsx:242]: I actually might have found the submarine  horn, when I get back home I will check if it works and I will implement it. 

Bell: 16
Submarine: 0a
Party: 17
Foghorn: 18

**Comment by sanderDijkxhoorn** [comment]
Own I just found out I messed something up and with big text it looks like this: 
image

**Comment by sanderDijkxhoorn** [comment]
Should I just make the buttons like this?
image

**Comment by sanderDijkxhoorn** [comment]
It works but it's not responsive for mobile.

**Comment by sanderDijkxhoorn** [comment]
> It works but it's not responsive for mobile.

image

**Comment by mjarkk** [comment]
Can we maybe shorten the names?
Like party horn can just be party

**Comment by sanderDijkxhoorn** [comment]
So the only thing I would have to add is it to automatically read the current set bell? (if possible)

**Comment by mjarkk** [comment]
It would be great if we can can also show the current bell sound!

**Comment by sanderDijkxhoorn** [comment]
> It would be great if we can can also show the current bell sound!

Alright, I will try my best 😄

**Pull request merged**